homeassistant/components/light/hue.py

```diff
@@ -147,6 +147,16 @@ def __init__(self, light_id, info, bridge, update_lights):
         self.bridge = bridge
         self.update_lights = update_lights
 
+        # Hue can control multiple type of lights
+        capability_map = {
+            'Dimmable light': ['bri'],
+            'Dimmable plug-in unit': ['bri'],
+            'Color light': ['bri', 'xy'],
+            'Extended color light': ['bri', 'xy', 'ct'],
+            'unknown': []}
+        self.capabilities = capability_map.get(
```

Comment:
`self.capabilities` will be set to 'unknown' now. The `get` method will return the second argument if supplied key is not found.
```python
        self.capabilities = {
            'Dimmable light': ['bri'],
            'Dimmable plug-in unit': ['bri'],
            'Color light': ['bri', 'xy'],
            'Extended color light': ['bri', 'xy', 'ct'],
        }.get(self.info['type'], [])
```